### PR TITLE
Products set order documentation

### PR DESCRIPTION
**Pull request by 00eadeyemi**
### Features 

- `Set Product Order` API documentation

[Delivers #132143811](https://www.pivotaltracker.com/story/show/132143811)

### CODE REVIEW

File: _includes/v4/products.md [hunk 1]
```diff
@@ -347,3 +347,49 @@ Name | Required | Type | Description | Displayed As
...
+Name | Required | Type | Description
+--- | --- | --- | ---
+`$.products` | Yes | array | An array containing the products you wish to order.
+`$.product_code` | Yes | string | The identifier for the product.
+`$.order` | Yes | int | The order for the product.
+
+
+#### Example
+
+```json
```
Comment (wnowicki) [inline _includes/v4/products.md]: I think that 

```json
{
    "products": {
        "AIN2-10": 0,
        "AIN3-6": 1,
        "IFC-09": 2,
        "IFC-06": 4
    }
}
```

makes more sense for me, but let's have team opinion.

Comment (EvanBarbour3) [inline _includes/v4/products.md]: This makes sense yeah, as it is the API for setting the order

Comment (srleach) [inline _includes/v4/products.md]: Yeah I agree. What's the behaviour if only a subset of loans are `POST`ed?

**Comment by wnowicki** [comment]
@00eadeyemi when change will be ready ?

**Comment by 00eadeyemi** [comment]
@wnowicki changes synced

**Comment by wnowicki** [comment]
- ok, structure is fine 👍
- please add some explanations about default product order, how it would work when we have semi ordered set and how to set product back to default order
- does always all products need to be send? 
- how this will work if we send again different product with order `1` will other previously set thing get lost?

@00eadeyemi

**Comment by 00eadeyemi** [comment]
@wnowicki please review, checked by @EvanBarbour3

**Comment by wnowicki** [comment]
- good
- one case:
Ive already set:
```json
{
    "AIN2-10": 0,
    "AIN3-6": 1,
    "IFC-09": 2,
    "IFC-06": 4
}
```

And then I'm calling:
```json
{
    "AIN2-10": 0,
    "IFC-06": 1
}
```

what will happen to `IFC-09` ?

@00eadeyemi

**Comment by 00eadeyemi** [comment]
@wnowicki `IFC-09` will still be 2 unless it is overwritten

**Comment by wnowicki** [comment]
- OK, approved
- we will wait with merging until changes release

**Pull request merged**